### PR TITLE
Completed the Messages and Sessions Component

### PR DESCRIPTION
**Pull request by edwardsmoses**

### CODE REVIEW

**Comment by edwardsmoses** [comment]
I was not too sure of the naming.
So, I used **Messages** as a _prefix_ for the first component, and **Sessions** as a _prefix_ for the second component.

File: src/Sessions/SessionDisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        onPress={(
+          selectedMessageStatus: SessionTypes.SessionMessageTypeEnum
+        ) => {
+          return toggleMessageStatus(selectedMessageStatus);
+        }}
+        currentMessageType={currentMessageStatus}
+      />
+      <SessionList
+        sessions={sessions.filter(
+          (m) => m.messageType === currentMessageStatus
```
Comment (mzruiz) [inline src/Sessions/SessionDisplay.tsx]: We don't need the messageType value for sessions. Instead, its **status** should determine where it goes. For instance, _Scheduled_ sessions should go under the **Scheduled** tab whereas the others go under the **History** tab.

**Review by mzruiz — CHANGES_REQUESTED** [review]
Looks good. Just the one change

File: src/Sessions/SessionDisplay.tsx (hunk 1 above)
Comment (edwardsmoses) [reply src/Sessions/SessionDisplay.tsx]: All right. I understand. 
I would update the types and the components.

**Review by mzruiz — APPROVED** [review]

**Pull request merged**